dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf

```diff
@@ -38,6 +38,35 @@ resource "kafka_topic" "uswitch_events_v1" {
   }
 }
 
+resource "kafka_topic" "uswitch_events_v2" {
+  name = "customer-proposition.uswitch.events.v2"
+
+  replication_factor = 3
+  partitions         = 15
+
+  # infinte retention
+  config = {
+    "remote.storage.enable" = "true"
+    "retention.bytes"       = "-1"
+    "retention.ms"          = "-1"
+    # keep data locally for 1 hour
+    "local.retention.ms" = "3600000"
+    # allow max 1 MB for a message
+    "max.message.bytes" = "1048576"
+    "compression.type"  = "zstd"
+    "cleanup.policy"    = "delete"
+  }
+}
+
+# tflint-ignore: terraform_naming_convention
```

Comment:
Please fix this name

Reply:
I can do if it's really necessary. We have used the same naming throughout this file, and we borrowed the structure of this from another namespaces files...

I'd rather not change it and have one odd module if possible

Reply:
Done 👍🏻